app/translations/en.json

```diff
@@ -14,6 +14,7 @@
   "donation": "Thanks for supporting Zingo!",
   "restarting": "Error connecting to the Server, the App will restart in a few seconds.",
   "change-privacy": "New privacy policy: ",
```

Comment:
can we have the format of this snackbar to be:
New privacy policy: High
Tap in an amount.....

it is all on one line and not easy to read

Reply:
i will PM you a screenshot